containerize/Containerfile

```diff
@@ -22,7 +22,7 @@ ARG FF_FILE=$DEPS_DIR/firefox.tar.bz2
 ARG TRIVY_FILE=$DEPS_DIR/trivy.tar.gz
 ARG KCTL_FILE=$DEPS_DIR/kubectl
 
-RUN microdnf install -y tar gzip bzip2 java-21-openjdk nodejs
+RUN microdnf install -y tar gzip bzip2 java-21-openjdk nodejs gtk3
```

Comment:
It seems that GTK3 is no longer included by default in UBI9-minimal, which causes the container build to fail with the following error:
```
[1/2] STEP 21/23: RUN . /cachi2/cachi2.env &&     echo "Validating Firefox version..." &&     /opt/firefox/firefox -version | grep "${FF_VERSION}" || (echo "Firefox version mismatch!" && exit 1) &&     echo "Firefox version validated: ${FF_VERSION}"
Validating Firefox version...
XPCOMGlueLoad error for file /opt/firefox/libmozgtk.so:
libgtk-3.so.0: cannot open shared object file: No such file or directory
Couldn't load XPCOM.
Firefox version mismatch!
subprocess exited with status 1
subprocess exited with status 1
```